src/main.ts

```diff
@@ -89,7 +89,7 @@ const getRedirectUrl = async (apiAddrs: string[]) => {
   const body = JSON.parse(value.body.substring(1));
   // TODO: remove backwards-compatible "EndorsedUI" after contract change.
   const referencedUI = body.current.ReferencedUI || body.current.EndorsedUI;
```

Comment:
drive-by: can we do this TODO now?

Reply:
Yea seems to work with mainnet, devnet, and emerynet, so removed this now.